### ISSUE TITLE
bg3 mod manager issue

### ISSUE THREAD

**Issue by moonstarr0414** [mkinfrared/baldurs-gate3-mod-manager#153]
## Your Environment

So my Baldur's Gate 3 Mod Manager keeps loading like I can see modification files but not my mods order and enable/disable meaning I can't add or remove mods. I'm new to modding so I don't really know what to do. I can still see the mods I had already installed on the bg3 game. I tried deleting and redownloading the mod manager, but no it's still not working.

- Manager Version : v1.5.0
- Link to problem mod : 
- Operating System and version : macOS, Sonoma 14.3

<img width="1512" alt="Screenshot 2024-02-14 at 4 34 23 PM" src="https://github.com/mkinfrared/baldurs-gate3-mod-manager/assets/160074211/723a3a9c-8213-4333-ae20-b0176bdbea3f">

**Comment by griffsian** [comment]
Hi, I also have the same issue on Sonoma 14.3.1.

**Comment by moonstarr0414** [comment]
> Hi, I also have the same issue on Sonoma 14.3.1.

did you find a solution?

**Comment by griffsian** [comment]
> > Hi, I also have the same issue on Sonoma 14.3.1.
> 
> did you find a solution?

Sadly not, still waiting for a response.

Edit: @moonstarr0414 My mods are now completely broken, even though Mac hasn't received Patch 6 yet. Not sure if this is because of the Mod Manager or another issue, it was working fine the day before yesterday. Will have to try re-installing the mods manually with the help of YouTube.

**Comment by moonstarr0414** [comment]
> > > Hi, I also have the same issue on Sonoma 14.3.1.
> > 
> > 
> > did you find a solution?
> 
> Sadly not, still waiting for a response.
> 
> Edit: @moonstarr0414 My mods are now completely broken, even though Mac hasn't received Patch 6 yet. Not sure if this is because of the Mod Manager or another issue, it was working fine the day before yesterday. Will have to try re-installing the mods manually with the help of YouTube.

That's weird. My mods still work fine. It's just BG3MM that's still not working. Maybe I'll also try to re-install them manually.

**Comment by helaenatargaryen** [comment]
> > > > Hi, I also have the same issue on Sonoma 14.3.1.
> > > 
> > > 
> > > did you find a solution?
> > 
> > 
> > Sadly not, still waiting for a response.
> > Edit: @moonstarr0414 My mods are now completely broken, even though Mac hasn't received Patch 6 yet. Not sure if this is because of the Mod Manager or another issue, it was working fine the day before yesterday. Will have to try re-installing the mods manually with the help of YouTube.
> 
> That's weird. My mods still work fine. It's just BG3MM that's still not working. Maybe I'll also try to re-install them manually.

hello! were you able to fix it? i'm having the same issue.

**Comment by moonstarr0414** [comment]
> > > > > Hi, I also have the same issue on Sonoma 14.3.1.
> > > > 
> > > > 
> > > > did you find a solution?
> > > 
> > > 
> > > Sadly not, still waiting for a response.
> > > Edit: @moonstarr0414 My mods are now completely broken, even though Mac hasn't received Patch 6 yet. Not sure if this is because of the Mod Manager or another issue, it was working fine the day before yesterday. Will have to try re-installing the mods manually with the help of YouTube.
> > 
> > 
> > That's weird. My mods still work fine. It's just BG3MM that's still not working. Maybe I'll also try to re-install them manually.
> 
> hello! were you able to fix it? i'm having the same issue.

Nope! Nothing has changed. I don't know what to do. it's getting annoying.

**Comment by axnito** [comment]
Hello! Has anyone here found out how to fix this? I'm struggling with the same issue

**Comment by BenjaminVanRyseghem** [comment]
I forked it, and fixed it.

I open a ticket to propose my changes to the author, but he didn't answer yet
